paymentsheet/src/main/java/com/stripe/android/paymentsheet/state/PaymentSheetLoader.kt

```diff
@@ -613,6 +618,16 @@ internal class DefaultPaymentSheetLoader @Inject constructor(
         }
     }
 
+    private fun isCvcRecollectionRequired(
```

Comment:
Can you rewrite this to be more readable? I'm thinking 2 booleans, with a return of 

`return deferredIntentRequiresCvcRecollection || paymentIntentRequiresCvcRecollection`